ddl/ddl.go

```diff
@@ -128,6 +128,8 @@ type DDL interface {
 	Stop() error
 	// RegisterEventCh registers event channel for ddl.
 	RegisterEventCh(chan<- *Event)
+	// SchemaVersionSyncer gets the schema version syncer.
+	SchemaVersionSyncer() *schemaVersionSyncer
```

Comment:
This function looks strange to be part of DDL interface.
You can make another interface instead of mix them together:

```
type SchemaVersionSyncer interface {
      Init()
      UpdateSelfVersion()
}
```

You can then mock the interface, use etcd or anything.

You can use the interface like this:

```
var ddl DDL
if syncer := ddl.(SchemaVersionSyncer); ok {
            // The same as you write syncer := ddl.SchemaVersionSyncer()
}
```